lib/widgets/movie.dart

```diff
@@ -0,0 +1,41 @@
+class Movie {
```

Comment:
Nice implementation the only suggestion here is that Movie is not a widget is a model, so you need to create a new folder name model and move this file

Reply:
solved